README.md

```diff
@@ -2,12 +2,11 @@
 # Example Client for Frequency
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-But it uses yarn instead of npm.
 
 ## Get started
-* Make sure [yarn is installed](https://classic.yarnpkg.com/en/docs/getting-started)
+* Make sure [npm is installed](https://docs.npmjs.com/cli/v9/configuring-npm/install)
 * Clone this repo
-* `yarn install`
+* `npm install`
 * Follow instructions for [building and starting Frequency](https://github.com/LibertyDSNP/frequency)
```

Comment:
(This could be a separate PR if you want) Could you add a note about matching versions?

I'm also tempted to say we should just suggest connecting to docker instant seal (easier version matching as well) instead of building.

Reply:
Updated the README with this info